README.md

```diff
@@ -26,7 +26,16 @@ Installation
 
 **Windows**
 
-[Detailed here](https://www.tuxemon.org/windows-install.html)
+Install the latest version of python 3 from [here](https://www.python.org/downloads/)
+
+Run:
+
+```cmd
+git clone https://github.com/Tuxemon/Tuxemon.git
+cd Tuxemon
+python -m pip install -r requirements.txt
```

Comment:
please change to `pip install -U -r requirements.txt`.  The `-U` flag is needed in the event that a dependency is installed but out of date.